tests/unit/conftest.py

```diff
@@ -72,15 +72,15 @@ def example_db_info():
 
 
 @pytest.fixture(scope="function")
-def setup_db_relation(harness, example_db_info):
+def setup_db_relation(harness: ops.testing.Harness, example_db_info: dict):
     """Yields a function that can be used to set up db relation.
 
     After calling the yielded function, a db relation will be set up. If ``db_info`` not provided
     during the invocation, example_db_info will be used as the relation data. Return a tuple of
     relation id and the relation data.
     """
 
-    def _setup_db_relation(db_info=None):
+    def _setup_db_relation(db_info: dict = None):
```

Comment:
Should this be `dict | None` or `typing.Union[dict, None]` for before Python 3.10?

Reply:
I removed this function argument after I found out that it's used anywhere. Thanks!